src/App.vue

```diff
@@ -3,3 +3,4 @@
 </template>
 
 <style src="./assets/css/app.sass" />
```

Comment:
SCSS is slightly better than SASS - it's more readable and the syntax is closer to regular CSS.
SCSS and SASS are the same engine, with just a different syntax. You just need to change the file extension from `sass` to `scss`